### ISSUE TITLE
Ability to rename a register similarly to IDA Pro

### ISSUE THREAD

**Issue by zankai2** [NationalSecurityAgency/ghidra#2396]
For example: Rename eax or s0 register for a certain memory regione

**Comment by ghizard** [comment]
Are you looking for something like #975 or something different?

**Comment by zankai2** [comment]
No.   It's nothing to do with the decompiler.   I want to rename eax in disassembly.
